### PR TITLE
Better organize upgrade setup instructions

### PR DESCRIPTION
**Pull request by frankbu**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

File: content/en/docs/setup/upgrade/gateways/index.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
 ---
-title: Configuring and managing gateways with multiple revisions
-description: Configuring and managing gateways with multiple revisions.
-weight: 25
-keywords: [kubernetes,upgrading]
+title: Managing Gateways with Multiple Revisions
```
Comment (ostromart) [inline content/en/docs/setup/upgrade/gateways/index.md:2]: Per https://github.com/istio/istio.io/pull/8299#issuecomment-717550680
let' mark this page experimental for now.

File: content/en/docs/setup/upgrade/gateways/index.md [hunk 2]
```diff
@@ -22,7 +23,7 @@ include `.` (dots).
 
 ## Istioctl
 
-This section covers installation and upgrade of separate control plane and gateways, using `istioctl`.
+This section covers installation and upgrade of a separate control plane and gateway using `istioctl`.
```
Comment (ostromart) [inline content/en/docs/setup/upgrade/gateways/index.md:26]: "covers the installation" ?

File: content/en/docs/setup/upgrade/gateways/index.md [hunk 3]
```diff
@@ -113,7 +117,7 @@ to install the Istio 1.8.1 control plane:
 
 ## Operator
 
-This section covers installation and upgrade of separate control plane and gateways, using the Istio operator.
+This section covers installation and upgrade of a separate control plane and gateway using the Istio operator.
```
Comment (ostromart) [inline content/en/docs/setup/upgrade/gateways/index.md:120]: "covers the installation"

File: content/en/docs/setup/upgrade/in-place/index.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+default values between Istio versions.
+
+The upgrade command can also perform a downgrade of Istio.
+
+See the [`istioctl` upgrade reference](/docs/reference/commands/istioctl/#istioctl-upgrade)
+for all the options provided by the `istioctl upgrade` command.
+
+## Upgrade prerequisites
+
+Ensure you meet these requirements before starting the upgrade process:
```
Comment (ostromart) [inline content/en/docs/setup/upgrade/in-place/index.md:22]: Ensure these requirements are met before starting the upgrade process:

File: content/en/docs/setup/upgrade/in-place/index.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+* Istio version is 1 minor version less than {{< istio_full_version >}}. For example, 1.6.0 or higher is required before you start the upgrade process to 1.7.0.
```
Comment (ostromart) [inline content/en/docs/setup/upgrade/in-place/index.md:24]: The installed Istio version is no more than one minor version less than...

File: content/en/docs/setup/upgrade/in-place/index.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+* Downgrade must be done using the `istioctl` binary version that
+corresponds to the Istio version that you intend to downgrade to.
+For example, if you are downgrading from Istio 1.7 to 1.6.5, use `istioctl`
+version 1.6.5.
+
+## Steps to downgrade to a lower Istio version
+
+You can use `istioctl upgrade` to downgrade to a lower version of Istio. Please
+notice that you need to use the `istioctl` binary corresponding to the lower
```
Comment (ostromart) [inline content/en/docs/setup/upgrade/in-place/index.md:96]: s/notice/note/

File: content/en/docs/setup/upgrade/in-place/index.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+version (e.g., 1.6.5). The process steps are
```
Comment (ostromart) [inline content/en/docs/setup/upgrade/in-place/index.md:97]: s/e.g.,/e.g./

File: content/en/docs/ops/configuration/traffic-management/network-topologies/index.md [hunk 8]
```diff
@@ -1,18 +1,18 @@
 ---
-title: Configuring Gateway Network Topology (Development)
+title: Configuring Gateway Network Topology [experimental]
 description: How to configure gateway network topology (Development).
```
Comment (ericvn) [inline content/en/docs/ops/configuration/traffic-management/network-topologies/index.md:3]: Remove development on this line as well.

Comment (frankbu) [inline content/en/docs/ops/configuration/traffic-management/network-topologies/index.md:3]:
```suggestion
description: How to configure gateway network topology.
```

**Review by ericvn — APPROVED** [review]

**Pull request merged**